src/unix/linux/system.rs

```diff
@@ -580,22 +580,43 @@ where
     }
 }
 
+fn read_table_key(filename: &str, target_key: &str, colsep: char) -> Option<u64> {
```

Comment:
`colsep` is always a whitespace, right? Then it can be removed and replaced in the code with the actual char.

Reply:
I see that in tests you change the value of `colsep`. Is there a reason for that?

Reply:
Thanks for the review! This function was meant to be more generic like `read_table` but if you want, I can make it more specific.

Reply:
@GuillaumeGomez do you see the `colsep` as a blocker? If so, I can definitely remove it.

Reply:
No it's fine, I just forgot about the PR, sorry about that. Taking a last look.